src/utilities/requests.ts

```diff
@@ -21,6 +21,7 @@ export async function reqExtension(
 ): Promise<ExtensionResponse> {
   const headers: HeadersInit = {
     Authorization: `Bearer ${user?.token ?? ''}`,
+    'x-hasura-role': (user as User)?.activeRole ?? '',
```

Comment:
Question: is `activeRole` always set for a User, or is missing if the user is using their default role just after login?

Reply:
From a quick check of the code it looks like it's always defined and we never defer to the users default role, but @duranb can hopefully clarify further!